python/treelite/gtil/gtil.py

```diff
@@ -26,34 +60,74 @@ def predict(model: Model, data: np.ndarray, nthread: int = -1, pred_margin: bool
         2D NumPy array, with which to run prediction
     nthread : :py:class:`int <python:int>`, optional
         Number of CPU cores to use in prediction. If <= 0, use all CPU cores.
-    pred_margin : :py:class:`bool <python:bool>`, optional
-        Whether to produce raw margin scores
+    pred_margin : bool, optional
+        Deprecated. Set predict_type="raw" instead.
+    predict_type : string, optional
+        One of the following:
+        * "default": Usual prediction method. Sum over trees and apply post-processing.
```

Comment:
Maybe it is better to also describe what the "default" method does through its name instead of calling it "default". Although I'm a bit at loss what a good name might be.

Reply:
I couldn't find a one-word summary of what this does. I'll just leave it as `default`.